aepp/utils.py

```diff
@@ -0,0 +1,25 @@
+from configparser import ConfigParser
+import os
+
+class Utils:
```

Comment:
Let's make all of these functions static so we don't need to instantiate `Utils` just to call them.